package/boot/kexec-tools/files/kdump.init

```diff
@@ -3,6 +3,8 @@
 START=41
 STOP=90
 
+BOOT_IMAGE=/boot/vmlinuz
```

Comment:
Isn't this variable?

Reply:
what do you think?
it just a variable in a boot-up script

Reply:
Try:

```
grep -r "KERNEL_NAME :=" include target
```

and you'll see what I'm talking about.

If this is just for x86, then try modifying `target/linux/x86/image/Makefile` to contain:

```
KERNEL_NAME := vmlinuz
```

and then try interpolating that string via a `sed` substitution.

Reply:
crashdump only work on x86

Reply:
it is for x86 only